src/main/java/org/jitsi/jigasi/SoundNotificationManager.java

```diff
@@ -386,12 +387,13 @@ private static void injectSoundFileInStream(
             System.arraycopy(
                 data, 0, rtp.getBuffer(), rtp.getPayloadOffset(), data.length);
 
+            long sleepTime = nSamples/48 - (System.currentTimeMillis() - timePushPreviousData);
```

Comment:
`timePushPreviousData` would have been set on the previous iteration *after* the `sleep()` call, so it doesn't take into account how long `sleep` took. Or am I missing something?

Reply:
Yes, that's true, but we count how long it took since we last sent a packet so we can sleep the rest of the time of that interval.